spec/requests/scim_v2/schemas_spec.rb

```diff
@@ -42,26 +42,93 @@
 
   describe "GET /scim_v2/Schemas" do
     context "with the feature flag enabled", with_flag: { scim_api: true } do
+      # rubocop:disable Layout/LineLength
       it "responds with supported schemas" do
         get "/scim_v2/Schemas", {}, headers
 
         response_body = JSON.parse(last_response.body)
         expect(response_body["totalResults"]).to eq(2)
         expect(response_body["schemas"]).to eq(["urn:ietf:params:scim:api:messages:2.0:ListResponse"])
-        expect(response_body["schemas"]).to eq(["urn:ietf:params:scim:api:messages:2.0:ListResponse"])
         group_schema = response_body["Resources"].find { |r| r["name"] == "Group" }
         user_schema = response_body["Resources"].find { |r| r["name"] == "User" }
-        external_id_schema = { "multiValued" => false,
-                               "required" => true,
-                               "caseExact" => true,
-                               "mutability" => "readWrite",
-                               "uniqueness" => "none",
-                               "returned" => "default",
-                               "name" => "externalId",
-                               "type" => "string" }
-        expect(group_schema["attributes"]).to include(external_id_schema)
-        expect(user_schema["attributes"]).to include(external_id_schema)
+
+        expect(group_schema).to eq(
+          "name" => "Group",
+          "id" => "urn:ietf:params:scim:schemas:core:2.0:Group",
+          "description" => "Represents a Group",
+          "meta" => { "resourceType" => "Schema",
+                      "location" => "http://test.host/scim_v2/Schemas?name=urn%3Aietf%3Aparams%3Ascim%3Aschemas%3Acore%3A2.0%3AGroup" },
+          "attributes" => [
+            { "multiValued" => false, "required" => true, "caseExact" => true, "mutability" => "readWrite", "uniqueness" => "none", "returned" => "default", "name" => "displayName", "type" => "string" },
```

Comment:
I see that we disabled the cop for line length before, but I believe that was done to allow writing the URL above in a single line. Since this is a hash, I'd say we should still write this in multiple lines for better readability. (the example has "too many lines" anyways, so we can exceed that even further xD)

Reply:
I changed formatting.